### PR TITLE
Correctly check env variable to initialize JwtManager

### PR DESCRIPTION
**Pull request by aschempp**

### CODE REVIEW

**Review by Toflar — APPROVED** [review]

File: manager-bundle/src/HttpKernel/ContaoKernel.php [hunk 1]
```diff
@@ -310,7 +310,7 @@ protected function initializeContainer(): void
         $container->set('contao_manager.plugin_loader', $this->getPluginLoader());
 
         // Set the JWT manager only if the debug mode has not been configured in env variables
-        if (!isset($_SERVER['APP_DEBUG']) && !isset($_SERVER['SYMFONY_DEBUG'])) {
+        if (!isset($_SERVER['APP_ENV']) && !isset($_SERVER['SYMFONY_ENV'])) {
```
Comment (leofeyer) [inline manager-bundle/src/HttpKernel/ContaoKernel.php:313]: Why did you change this although the comment says to check the debug mode?

Comment (aschempp) [reply manager-bundle/src/HttpKernel/ContaoKernel.php:313]: `APP_DEBUG` was the initial approach to switching the environment. We've changed it because `APP_ENV` is what Symfony primarily listens to, but forgot to change this accordingly.

**Comment by leofeyer** [comment]
Thank you @aschempp.
